docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="HPCC_Azure_deployment">
+  <title>Azure Deployment (Development, Testing, and Production)</title>
+
+  <para>The following section and commands are all intended to be general
+  enough to apply for most all common Azure subscriptions. This section is
+  intended to be logical enough and relatively intuitive enough to determine
+  what if any changes or modifications are to comply with any possible
+  conflicts or discrepancies in these following pages.</para>
+
+  <sect1 id="Using_Azure_SECT1" role="nobrk">
+    <title>Using Azure</title>
+
+    <para>Though there are many ways to interact with Azure, this section will
+    focus on doing so using the Azure cloud shell command line
+    interface.</para>
+
+    <para>The major advantage to using the cloud shell is that It will also
+    have the other prerequisites installed for you. Otherwise in addition to
+    installing the Azure CLI you would also need to install the other
```

Comment:
remove redundant sentence. 
Otherwise in addition to installing the Azure CLI you would also need to install the the
other prerequisites separately.